.github/workflows/ci.yml

```
@@ -13,13 +13,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: ['1.19', '1.20', '1.21']
+        go-version: ['1.21', '1.22', '1.23']
```

I would use


```suggestion
        go-version: ['1.21', 'oldstable', 'stable']
```

For now, it would be 1.21+

When Go 1.24 will be out, the action will then use 1.21, 1.23, 1.24